inlong-manager/manager-web/src/main/resources/application-prod.properties

```diff
@@ -54,19 +54,10 @@ cluster.tube.master=127.0.0.1:8000,127.0.0.1:8010
 # Tube cluster ID
 cluster.tube.clusterId=1
 
-# Push configuration to the path on ZooKeeper
-cluster.zk.url=127.0.0.1:2181
-cluster.zk.root=inlong_hive
-
-# Application name in Sort
-sort.appName=inlong_app
-
 # Pulsar admin URL
 pulsar.adminUrl=http://127.0.0.1:8080,127.0.0.2:8080,127.0.0.3:8080
```

Comment:
Please check if those Pulsar parameters are used. If not, it is recommended to delete it.